tests/foreman/api/test_host.py

```diff
@@ -391,45 +467,59 @@ def test_positive_create_with_compresource(self):
         host = entities.Host(
             compute_resource=compresource, location=loc, organization=org
         ).create()
-        self.assertEqual(host.compute_resource.read().name, compresource.name)
+        assert host.compute_resource.read().name == compresource.name
+        new_compresource = entities.LibvirtComputeResource(
```

Comment:
check if same available in global fixtures else worth creating there ?

Reply:
DONE.